lavorare-su-nebulab/blog.md

```diff
@@ -0,0 +1,35 @@
+# Blog
+
+[Il nostro blog](https://nebulab.it/blog/) è gestito da Shana ed è uno dei modi con cui contribuiamo 
+al panorama OSS, nonché un modo per farci conoscere da potenziali clienti. Chiunque può scrivere sul 
+blog e tutti sono incoraggiati a farlo! Scrivere è anche un ottimo modo per capire gli argomenti che 
+stiamo studiando e i problemi che stiamo affrontando, dunque il risutlato di un progetto del venerdì 
+potrebbe essere un articolo sul blog.
+
+## Scrivere un post
+
+Scrivere un post è un processo che si compone di quattro fasi ed è completamente gestito in un
+[progetto GitHub](https://github.com/nebulab/nebulab/projects/3):
+
+1. **Proposta:** se vuoi proporre un articolo, apri una issue e aggiungila al backlog del progetto 
+[Blogposts](https://github.com/nebulab/nebulab/projects/3) su GitHub. Nella issue inserisci qualche
+dettaglio su ciò di cui vorresti parlare e su come vuoi affrontarlo. Pinga tutte le persone da cui 
+vorresti del feedback o chiedi in #nebulab.
+2. **Accettazione:** una volta che sono stati raccolti abbastanza elementi per scrivere un articolo,
+lascia una nota per Shana, che provederà a spostarlo in ToDo. A questo punto puoi iniziare a 
+lavorarci! 
+3. **Stesura:** scrivi l'articolo seguendo [le linee guida](https://github.com/nebulab/nebulab/blob/master/source/blog/posts/2050-01-01-blogpost-template.html.markdown)
+nel repository. Se hai qualche dubbio o hai bisogno di aiuto, scrivi in #nebulab e qualcuno sarà
+sicuramente felice di darti una mano. 
+4. **Feedbcak:** una volta che hai finito la stesura iniziale, cerca di raccogliere feedback sul tuo 
+articolo da più persone possibili. La tua PR deve essere approvata da almeno due persone per poter
+essere mergiata. 
```

Comment:
Manca un punto: review dell'articolo da parte di Shana che suggerirà modifiche per la SEO.

Reply:
Quoto Matteo G..

Inoltre vorrei essere coinvolta nella fase di accettazione. Quando si arriva al punto in cui l'argomento è stato sviscerato, analizzato e riorganizzato a livello tecnico in modo soddisfacente, e che quindi è pronto per la stesura, sarebbe utile per me avere un recap dell'argomento, dello stile che verrà utilizzato, sapere a chi è rivolto, perchè è interessante, ecc... In questo modo ho il tempo sia di dare dei feedback prima di cominciare sia per fare ricerche SEO sull'argomento per l'eventuale ottimizzazione.

Reply:
@ShanaRem @matteogalliani aggiungerò dettagli sulla review SEO. @ShanaRem sei già tu la responsabile dell'accettazione, se guardi il primo step. O volevi che aggiungessi qualcosa in particolare?